CircuitPython_Essentials/CircuitPython_I2C_Scan.py

```diff
@@ -1,15 +1,24 @@
 # CircuitPython demo - I2C scan
+# 
+# If you run this and it seems to hang, try manually unlocking
+# your I2C bus from the REPL with
+# ` >>> import board `
+# ` >>> board.I2C().unlock() `
```

Comment:
I do not believe the backticks are necessary here as it is already in a comment. If you included them for a reason, please let me know what it was.

Reply:
done, thought they might be necessary for auto gen docs to show as code, but in reality this code isn't used that way.